Document first authors

dewittpe opened the pull request:
@tdbennett, I've added notes for equal contribution for you and Nelson, along with a similar not for Seth and I.  Please review.

There is one exception.  I didn't add any note within the Description section of the DESCRIPTION file.  To add the note there I would think a parenthetical would be better than just astriks.

I've opted to use the term "contributed equally" instead of co-first-author or co-second-author to be consistent with the language used by JAMA.

dewittpe committed:
Update all references to the data paper with a note for equal contributions between Sanchez-Pinto and Bennet; DeWitt and Russell.

Prior to this change all the references where just copied from the "Cite
This" provided on the JAMA website.

This change add Russell explicitly and asstricks and a note about the
contributions.  The language is copied form the JAMA paper as well.

There is one exception - the DESCRIPTION file does not have these
indicators for contribution levels.

re #7

dewittpe committed:
use the \doi{} macro in references

tdbennett commented:
Hi Peter - it looks like the mentions of the manuscripts in the README still need to be updated. thanks

dewittpe committed:
Add &ast; in the README to show equal contribution by authors on the Dev paper

re #7, #8

dewittpe committed:
correct spelling of equally

dewittpe commented:
@tdbennett I've added asterisks to our names in the README

dewittpe commented on README.md:
```diff
@@ -15,9 +15,11 @@
 described in:
 
-* ["Development and Validation of the Phoenix Criteria for Pediatric Sepsis and Septic Shock"](doi:10.1001/jama.2024.0196) by Sanchez-Pinto, Bennett, DeWitt, Russell et.al. (2024)
+* ["Development and Validation of the Phoenix Criteria for Pediatric Sepsis and Septic Shock"](doi:10.1001/jama.2024.0196) by Sanchez-Pinto&ast;, Bennett&ast;, DeWitt&ast;&ast;, Russell&ast;&ast; et al. (2024)
```
@tdbennett this should render well.  I used the html code `&ast;` to avoid markdown formatting conflicts

dewittpe committed:
update DESCRIPTION re #7 and #8

dewittpe committed:
Merge branch 'main' into document-first-authors

dewittpe committed:
Improving documentation of reference materials

Refactor how the two JAMA papers are cited - all the authors and equal
contribution notes are in the phoenix-package documentation with links
and directions to review the details there in all the other function
manual files.   The CITATION file as been extended to include notes on
the two JAMA papers and instructions on how to cite the R package.

re #7, #8

dewittpe committed:
Merge branch 'main' into document-first-authors
